src/sidenav/sidenav-item.styles.tsx

```diff
@@ -0,0 +1,77 @@
+import styled, { css } from "styled-components";
+import { Text } from "../text";
+import { Color } from "../color";
+import { Button } from "../button";
+
+interface StyleProps {
+    $highlight: boolean;
+}
+
+export const Container = styled.li``;
+
+export const DefaultButton = styled(Button.Default)<StyleProps>`
```

Comment:
If I use styled.button, it will expand the css bit more. Need to define background colours, text colours, alignments and behaviours. Can we stick with DS button?

Reply:
ok. can stick with this first. but you'll need to override the fixed `height` of 3rem